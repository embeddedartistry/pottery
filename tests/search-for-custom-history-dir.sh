pottery init history-log
pottery post First Record
mkdir subdir
cd subdir
pottery post Second Record
find . -print
cd ..
find . -print
