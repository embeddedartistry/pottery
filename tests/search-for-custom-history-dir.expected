pottery init history-log
pottery post First Record
mkdir subdir
cd subdir
pottery post Second Record
find . -print
.
cd ..
find . -print
.
./.project-history-dir
./history-log
./history-log/1992
./history-log/1992/1992-01
./history-log/1992/1992-01/1992-01-12T16:34:23Z.md
./subdir
